.gitignore

```diff
@@ -8,5 +8,5 @@ test-results
 yarn-error.log
 __pycache__
 *.terraform.lock.hcl
-*.terraform
+*.terraform*
```

Comment:
Is there any downside to adding these `*.terraform*` files to git?
Or is there any benefit to adding them to git for a team size greater than one person 🙂  ?

Reply:
as far as what i understood about terraform is that.
`terraform.tfstate, terraform.tfstate.backup ` files and `.terraform ` directory
these files contain the terraform state specific of a specific environment there by didn't consider having them in the repository

Reply:
Generally, Terraform code and the environment it's deployed to is 1:1, so it's a reasonable practice to store the terraform state files in git. If multiple team members are collaborating, then you'll need to share the state, so things are kept in sync. For simple things, you can use git for this.
That said, some Terraform resources store sensitive data in the state, which then becomes inappropriate to store in git and so secure _remote_ state becomes a better option. Terraform Cloud solves this and a lot of other problems for us.

Reply:
Cool will keep it in mind from now on thanks